pkg/sanity/sanity.go

```diff
@@ -252,8 +252,10 @@ func (sc *SanityContext) Setup() {
 
 func (sc *SanityContext) Teardown() {
 	// Delete the created paths if any.
-	removeMountTargetLocation(sc.TargetPath, sc.Config.RemoveTargetPathCmd, sc.Config.RemoveTargetPath, sc.Config.RemovePathCmdTimeout)
-	removeMountTargetLocation(sc.StagingPath, sc.Config.RemoveStagingPathCmd, sc.Config.RemoveStagingPath, sc.Config.RemovePathCmdTimeout)
+	err := removeMountTargetLocation(sc.TargetPath, sc.Config.RemoveTargetPathCmd, sc.Config.RemoveTargetPath, sc.Config.RemovePathCmdTimeout)
+	Expect(err).NotTo(HaveOccurred(), "failed to delete target directory %s", sc.TargetPath)
```

Comment:
If the check fails, `Teardown` will abort and skip the next `removeMountTargetLocation`. It would be nicer if it did both calls, then checked that both errors are nil.

Reply:
```go
err1 := removeMountTargetLocation(sc.TargetPath, sc.Config.RemoveTargetPathCmd, sc.Config.RemoveTargetPath, sc.Config.RemovePathCmdTimeout)
err2 := removeMountTargetLocation(sc.StagingPath, sc.Config.RemoveStagingPathCmd, sc.Config.RemoveStagingPath, sc.Config.RemovePathCmdTimeout)
Expect(err1).NotTo(HaveOccurred(), "failed to delete target directory %s", sc.TargetPath)
Expect(err2).NotTo(HaveOccurred(), "failed to delete staging directory %s", sc.StagingPath)
```
Do you mean that I need to modify it to be like this?

Reply:
Yes, or have a single assertion that both errors are nil. What you have may be simpler.

Reply:
done. Now it can show all failures.